codegen/arrai/svc_types.arrai

```diff
@@ -69,6 +69,37 @@ let orderedTypes = \types
                         >> $`${go.name(.@item('name').s)} ${go.type(.@item('type'))}`
                     ::\i}
                 }
+
+                ${
+                let opParams = ep('restParams')?('queryParam')?.a:{} where $`${sysl.isPtr(.@item('type'))}`;
```

Comment:
No need for the expression-string wrapper.
```suggestion
                let opParams = ep('restParams')?('queryParam')?.a:{} where sysl.isPtr(.@item('type'));
```

Reply:
ok